xen/drivers/passthrough/iommu.c

```diff
@@ -273,21 +290,37 @@ int iommu_map_page(struct domain *d, unsigned long gfn, unsigned long mfn,
     return rc;
 }
 
-int iommu_unmap_page(struct domain *d, unsigned long gfn)
+int iommu_unmap_pages(struct domain *d, unsigned long gfn,
```

Comment:
same as above: size_t/pfn_t - I assume this is because hd->platform_ops->map_page API